no-computer.md

```diff
@@ -11,41 +11,87 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+_The idea is that the wooden blocks you put together to make patterns are a
+DSL. (http://www.hand2mind.com/item/wooden-pattern-blocks-1cm-set-of-250/1218?gclid=CJXS3N2Dos8CFYmCfgodAzcAVg&gclsrc=aw.ds)
+The need met is learning about patterns and shapes. Additionally, there is
+a need of visualizing a pattern before using it. Before wooden blocks, there
+was a difficult experience of seeing how the shapes could fit together and
+understanding how angles worked in different shapes. These blocks give a
+tactile, visual way of learning about patterns and angles._
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+_This language is appropriate because it does not involve drawing, which
+can be inprecise, or imagining, which can leave people unsatisfied. It 
+addresses the need by giving them wooden blocks, which have all been cut
+to be certain shapes. This lets them interact with the blocks and try all
+different types of blocks to see which ones fit._
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+_I love this idea because there are only so many ways that the
+blocks can interact with each other that does not leave any blank space
+in between them. It's such an interesting way to introduce geometry and
```

Comment:
This is an interesting restriction. I initially thought that the blank spaces can form a pattern itself, but I see how ensuring that the space is filled is an interesting challenge that could lead to a cool pattern.